contracts/Content/UniqueContent.sol

```diff
@@ -23,18 +33,13 @@ contract UniqueContent is IUniqueContent, IMultipleRoyalties, MultipleRoyalties,
     function mint(LibAsset.UniqueAssetCreateData memory _data) external override {
         require((IERC1155Upgradeable(_data.contentAddress).balanceOf(_msgSender(), _data.tokenId)) >= 1, "You must have the original item in your wallet");
         (, uint256 _originalRoyaltyRate) = IERC2981Upgradeable(_data.contentAddress).royaltyInfo(_data.tokenId, 1e6);
-        require(_verifyRoyalties(_data.royaltyReceivers, _data.royaltyRates, _originalRoyaltyRate), "The royalties entered are invalid");
+        require(uniqueContentStorage.verifyRoyalties(_data.royaltyReceivers, _data.royaltyRates, _originalRoyaltyRate), "The royalties entered are invalid");
```

Comment:
Actually, you can move verifyRoyalties (and _verifyRoyalties) out of MultipleRoyalties into LibRoyalty.sol

You can just end up calling LibRoyalty.verifyMultipleRoyalties(recievers, rates, originalroyatlyrate) in UniqueContent.sol and UniqueContentStorage(). That should lower the transaction cost and since it's only making one call instead of calling a function in UniqueContentStorage and forcing you to expose an additional function.